### PR TITLE
Web/web templates

### PR DESCRIPTION
**Pull request by chenc-allegheny**
# Description of Pull Request
I made another branch which is identical to current web/.....authentication. too many people is working on it now. I will close the #198, and focusing on fixing the current linting problems and get the templates merged.

    1. Remove bootstraps
    2. rendered templates
    3. GitHub/google authentication with login buttons on the home page
    4. monkeytype is for type-hint
  

Fixes #

## Type of Change

- [ ] Bug fix
- [ ] Breaking change
- [x] New feature
- [ ] Documentation update
- [ ] Other

### CODE REVIEW

File: gatorgrouper/static/gatorgrouper/LICENSE [hunk 1]
```diff
@@ -0,0 +1,22 @@
+The MIT License (MIT)
```
Comment (Michionlion) [inline gatorgrouper/static/gatorgrouper/LICENSE]: What is this license for?

File: gatorgrouper/urls.py [hunk 2]
```diff
@@ -1,9 +1,19 @@
...
 # from django.conf.urls.static import static
 from . import views
 
-urlpatterns = [re_path(r"^$", views.index, name="index")]
+urlpatterns = [
+    path("index", views.index, name="index"),
+    path(
+        "", views.home, name="Gtorgrouper-home"
+    ),  # first attribute is a space, meaning homepage
+    path("classes", views.create_classes, name="Gtorgrouper-classes"),
```
Comment (Michionlion) [inline gatorgrouper/urls.py]: There are two misspellings of "Gatorgrouper" here.

Comment (chenc-allegheny) [reply gatorgrouper/urls.py]: saijin don't leave! I am here I will do exactly what you tell me to do!

File: gatorgrouper/static/gatorgrouper/LICENSE (hunk 1 above)
Comment (chenc-allegheny) [reply gatorgrouper/static/gatorgrouper/LICENSE]: I believe that comes with the bootstrap or the scss that we had, I am getting rid of it now

**Comment by chenc-allegheny** [comment]
I had # pylint: disable=bad-continuation in display.py and group_random.py and group_scoring.py because black prefers four spaces at the beginning of the line, but pylint is asking for eight.

**Comment by Michionlion** [comment]
Looks good! When @Lancasterwu and @chenc-allegheny give the okay, I'll merge this.

**Comment by chenc-allegheny** [comment]
I can't assign myself as a reviewer(?) I will go hunt someone down in Alden

**Comment by Michionlion** [comment]
> I can't assign myself as a reviewer(?) I will go hunt someone down in Alden

I don't need another reviewer, just for you to say that it's good :)

**Comment by chenc-allegheny** [comment]
I's GOOOOOOOOOOD so GOOOOOOOOOOOOOOD

**Pull request merged**